src/Pdf.php

```diff
@@ -125,6 +125,31 @@ public function saveImage($pathToImage)
     }
 
     /**
+     * Save the file as images to the given path.
+     *
+     * @param string $storeDirectory
+     * @param string $file_name     
+     *
+     * @return array $files the full path to the newly created images.
+     */
+    public function extractAllPagesAsImages($storeDirectory, $fileName)
+    {
+        $pages = $this->getNumberOfPages();
+        $files = [];
+        for($i =1; $i <= $pages; $i++)
```

Comment:
A for loop will do the job, but I think
`foreach(range(1,$numberOfPages) as $pageNumber) {` is more readable.